notebooks/ViewingPHAResponses.ipynb

```diff
@@ -0,0 +1,10127 @@
+       "      </g>\n",
+       "     </g>\n",
+       "     <g id=\"text_2\">\n",
+       "      <!-- 400 -->\n",
+       "      <g transform=\"translate(179.589964 322.182437) scale(0.1 -0.1)\">\n",
+       "       <defs>\n",
+       "        <path id=\"DejaVuSans-34\" d=\"M 2419 4116 \n",
+       "L 825 1625 \n",
+       "L 2419 1625 \n",
+       "L 2419 4116 \n",
+       "z\n",
+       "M 2253 4666 \n",
+       "L 3047 4666 \n",
+       "L 3047 1625 \n",
+       "L 3713 1625 \n",
+       "L 3713 1100 \n",
+       "L 3047 1100 \n",
+       "L 3047 0 \n",
+       "L 2419 0 \n",
+       "L 2419 1100 \n",
+       "L 313 1100 \n",
+       "L 313 1709 \n",
+       "L 2253 4666 \n",
+       "z\n",
+       "\" transform=\"scale(0.015625)\"/>\n",
+       "       </defs>\n",
+       "       <use xlink:href=\"#DejaVuSans-34\"/>\n",
+       "       <use xlink:href=\"#DejaVuSans-30\" x=\"63.623047\"/>\n",
+       "       <use xlink:href=\"#DejaVuSans-30\" x=\"127.246094\"/>\n",
+       "      </g>\n",
+       "     </g>\n",
+       "    </g>\n",
+       "    <g id=\"xtick_3\">\n",
+       "     <g id=\"line2d_3\">\n",
+       "      <g>\n",
+       "       <use xlink:href=\"#mcabf5abb21\" x=\"238.505143\" y=\"307.584\" style=\"stroke: #000000; stroke-width: 0.8\"/>\n",
+       "      </g>\n",
+       "     </g>\n",
+       "     <g id=\"text_3\">\n",
+       "      <!-- 600 -->\n",
+       "      <g transform=\"translate(228.961393 322.182437) scale(0.1 -0.1)\">\n",
+       "       <defs>\n",
+       "        <path id=\"DejaVuSans-36\" d=\"M 2113 2584 \n",
+       "Q 1688 2584 1439 2293 \n",
+       "Q 1191 2003 1191 1497 \n",
+       "Q 1191 994 1439 701 \n",
+       "Q 1688 409 2113 409 \n",
+       "Q 2538 409 2786 701 \n",
+       "Q 3034 994 3034 1497 \n",
+       "Q 3034 2003 2786 2293 \n",
+       "Q 2538 2584 2113 2584 \n",
+       "z\n",
+       "M 3366 4563 \n",
+       "L 3366 3988 \n",
+       "Q 3128 4100 2886 4159 \n",
+       "Q 2644 4219 2406 4219 \n",
+       "Q 1781 4219 1451 3797 \n",
+       "Q 1122 3375 1075 2522 \n",
+       "Q 1259 2794 1537 2939 \n",
+       "Q 1816 3084 2150 3084 \n",
+       "Q 2853 3084 3261 2657 \n",
+       "Q 3669 2231 3669 1497 \n",
+       "Q 3669 778 3244 343 \n",
+       "Q 2819 -91 2113 -91 \n",
+       "Q 1303 -91 875 529 \n",
+       "Q 447 1150 447 2328 \n",
+       "Q 447 3434 972 4092 \n",
+       "Q 1497 4750 2381 4750 \n",
+       "Q 2619 4750 2861 4703 \n",
+       "Q 3103 4656 3366 4563 \n",
+       "z\n",
+       "\" transform=\"scale(0.015625)\"/>\n",
+       "       </defs>\n",
+       "       <use xlink:href=\"#DejaVuSans-36\"/>\n",
+       "       <use xlink:href=\"#DejaVuSans-30\" x=\"63.623047\"/>\n",
+       "       <use xlink:href=\"#DejaVuSans-30\" x=\"127.246094\"/>\n",
+       "      </g>\n",
+       "     </g>\n",
+       "    </g>\n",
+       "    <g id=\"xtick_4\">\n",
+       "     <g id=\"lin
```

Comment:
```suggestion
    "is now not a rate. In fact, because the ARF has units of cm$^2$, the model evaluation will calculate the number of photons per bin."
```

Reply:
[aain what is going on]